src/uu/wc/src/wc.rs

```diff
@@ -167,7 +167,7 @@ impl<'a> Inputs<'a> {
                     None => Ok(Self::Files0From(input)),
                 }
             }
-            (Some(_), Some(_)) => Err(WcError::FilesDisabled.into()),
+            (Some(mut files), Some(_)) => Err(WcError::files_disabled(files.next()).into()),
```

Comment:
I think it would be cleaner to do the `unwrap()` here because here it is obvious why it is safe to do. In `files_disabled` the context is missing.

Reply:
@cakebaker updated